.travis.yml

```diff
@@ -3,16 +3,26 @@ install:
 
 matrix:
   include:
-    - os: linux
-      # We use Docker → we need sudo.
-      sudo: required
-      services:
-        - docker
-      script:
-        - ci/build.sh
-    - os: osx
-      script:
-        - ci/macOS.sh
+  - os: linux
+    sudo: required
+    services:
+    - docker
+    script:
+    - ci/build.sh
+  - os: osx
+    script:
+    - ci/macOS.sh
```

Comment:
`build-macos.sh`?

Reply:
I want to refactor it separately, it's a little messy atm :(